test-plugins/org.yakindu.sct.model.resource.test/src/org/yakindu/sct/model/resource/test/AllTests.java

```diff
@@ -0,0 +1,21 @@
+/** 
+ * Copyright (c) 2016 committers of YAKINDU and others. 
+ * All rights reserved. This program and the accompanying materials 
+ * are made available under the terms of the Eclipse Public License v1.0 
+ * which accompanies this distribution, and is available at 
+ * http://www.eclipse.org/legal/epl-v10.html 
+ * Contributors:
+ * committers of YAKINDU - initial API and implementation
+ *
+*/
+package org.yakindu.sct.model.resource.test;
+
+import org.junit.runner.RunWith;
+import org.junit.runners.Suite;
+import org.junit.runners.Suite.SuiteClasses;
+
+@RunWith(Suite.class)
+@SuiteClasses({DomainSpecificTaskFinderTest.class})
+public class AllTests {
```

Comment:
Why is the AllTests empty? The Jenkins build will fail for test plugins that do not contain any tests